generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.reducer.ts.ejs

```diff
@@ -85,7 +85,6 @@ export default (state: UserManagementState = initialState, action): UserManageme
     case SUCCESS(ACTION_TYPES.FETCH_ROLES):
       return {
         ...state,
-        loading: false,
```

Comment:
If you dont reset this back it meand yiur loading icon will keep appearing if you try to rely on this variable. I don't get how this solves the issue, you might be suppressing the real issue by doing this, please check

Reply:
@deepu105, I am not a react expert, but as I understood the render() will kick in as soon as props are updated by the reducer, which means we really don't need the loading boolean, as soon as getUser and getRoles return props, they are supposed to dynamically be updated in the DOM. What I found in my testing is that, the AvInput checkbox refuses to update when DOM is re-rendered. I changed the checkbox to a text input, and it updates to true/false based on the value of the user.activated field on re-rendering. The text input updates fine on re-render when relying on the loading boolean or without it. 

If we were to rely on the loading boolean, then don't we need to wait for both the getRoles and getUser calls to complete and then render the page? But then we need to return two diff boolean props.